Resources/Prototypes/_Scp/Maps/scp_complex_compact.yml

```
@@ -23,6 +24,7 @@
             Scp096: [ 1, 1 ]
             Scp939: [ 1, 1 ]
             Scp173: [ 1, 1 ]
+            #Scp106: [ 0, 0 ] # Под него нету камеры на Компакте
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)